tutorials/strapi-app-serverless-containers-sqldb/index.mdx

```diff
@@ -0,0 +1,833 @@
+    scw container container create name="my-strapi-blog" \
+    namespace-id=$CONTAINER_NAMESPACE_ID \
+    registry-image=$REGISTRY_ENDPOINT/my-strapi-blog:latest \
+    port=1337 environment-variables.DATABASE_HOST=$DATABASE_HOST \
+    environment-variables.DATABASE_PORT=$DATABASE_PORT \
+    environment-variables.DATABASE_NAME=$DATABASE_NAME \
+    environment-variables.DATABASE_CLIENT=$DATABASE_CLIENT \
+    environment-variables.DATABASE_SSL=$DATABASE_SSL \
+    environment-variables.DATABASE_USERNAME=$DATABASE_USERNAME \
+    secret-environment-variables.0.key='DATABASE_PASSWORD' \
+    secret-environment-variables.0.value=$DATABASE_PASSWORD \
+    secret-environment-variables.1.key='APP_KEYS' \
+    secret-environment-variables.1.value=$APP_KEYS \
+    secret-environment-variables.2.key='API_TOKEN_SALT' \
+    secret-environment-variables.2.value=$API_TOKEN_SALT \
+    secret-environment-variables.3.key='ADMIN_JWT_SECRET' \
+    secret-environment-variables.3.value=$ADMIN_JWT_SECRET \
+    secret-environment-variables.4.key='TRANSFER_TOKEN_SALT' \
+    secret-environment-variables.4.value=$TRANSFER_TOKEN_SALT \
+    secret-environment-variables.5.key='JWT_SECRET' \
+    secret-environment-variables.5.value=$JWT_SECRET
+    ```
+
+    <Message type="tip">
+    The first deployment can take a few minutes. You can check the deployment status with the following command:
+    ```sh
+    scw container container list name=my-strapi-blog -o human=Status
+    ```
+    When the status appears as `ready`, you can access the Strapi Administration Panel via your browser.
+    </Message>
+  
+3. Copy the endpoint URL displayed next to the `DomainName` property, and paste it into your browser. The main Strapi page displays. Click on "Open the administration" or add `/admin` to your browser URL to access Strapi Administration Panel.
+
+4. (Optional) You can check that Strapi APIs are working with the following command, or  by accessing `https://{container_url}/api/articles` in your browser:
+    ```bash
+    curl -X GET https://{container_url}/api/articles | jq
+    ```
+
+    The available articles should display in JSON format:
+    ```json
+    {
+      "data": [
+      {
+        "id": 1,
+        "attributes": {
+          "title": "What's inside a Black Hole",
+          "description": "Maybe the answer is in this article, or not...",
+          "slug": "what-s-inside-a-black-hole",
+          "createdAt": "2024-04-16T15:49:14.525Z",
+          "updatedAt": "2024-04-16T15:49:14.525Z",
+          "publishedAt": "2024-04-16T15:49:14.457Z"
+        }
+      },
+        (...)
+      ]
+    }
+    ```
+
+Congratulations! Your Strapi headless CMS for a blog is now online. You can already edit stored content, or consume it from a frontend application.
+
+Moreover, your textual content data is already stored in serverless storage, so that your full application only runs when you connect to the administration panel or perform requests to Strapi API.
```

Comment:
```suggestion
Moreover, your textual content data is already stored in serverless storage, so that your full application only runs when you connect to the administration panel or perform requests to the Strapi API.
```